api/Document.json

```diff
@@ -7269,12 +7333,24 @@
               "version_added": "11",
               "alternative_name": "onmsfullscreenerror"
             },
-            "opera": {
-              "version_added": true
-            },
-            "opera_android": {
-              "version_added": true
-            },
+            "opera": [
+              {
+                "version_added": "58"
+              },
+              {
+                "version_added": "32",
```

Comment:
This is also wrong per http://mdn-bcd-collector.appspot.com/tests/api/Document/onwebkitfullscreenerror. It wasn't added at the same time as `onwebkitfullscreenchange`, but before Chrome 45 at least.